tests/framework/installer/ceph_installer.go

```diff
@@ -140,6 +152,12 @@ func (h *CephInstaller) CreateRookOperatorViaHelm(namespace, chartSettings strin
 		return err
 	}
 
+	err = h.startAdmissionControllers(namespace)
```

Comment:
You're configuring the admission controller after the helm chart is installed, which means the operator will already be running. You need to configure the admission controller before launching the helm chart.